### DISCUSSION TITLE
Loadfrom JSON em um novo fdmemtable

### DISCUSSION THREAD

**Discussion by FranziniDev** [viniciussanchez/dataset-serialize#234] [General]
Estou criando o fdmemtable não atribuo nenhum campo a ele, executo o LoadFromJson, ele cria os campos no fdmemtable, e tambem tras os devidos valores, porem, se neste Json contiver algum campo que contenha no nome "_"   como por exemplo "Taxa_entrega" , o valor não é carregado, ficando em branco o valor.

**Comment by FranziniDev** [comment]
Consegui resolver inserindo a configuração : 
TDataSetSerializeConfig.GetInstance.CaseNameDefinition := TCaseNameDefinition.cndNone; 

Não tenho certeza que é a melhor solução, mas enfim, funcionou como eu precisava.

**Reply by viniciussanchez** [reply]
Sim, é isso mesmo... você pode informar qual o padrão do JSON através dessa configuração.
